### PR TITLE
Create ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by janh

Ticket: [MTM-55762](https://cumulocity.atlassian.net/browse/MTM-55762)
Original PR: [4776](https://github.softwareag.com/IOTA/cumulocity-ui/pull/4776)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: asset selector showing by default the selected element (#4776)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Asset selector shows the selected element by default
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55762
+version: 1020.0.0
+---
+asset selector showing by default the selected element (#4776)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, the asset selector did not show the currently selected element by default, requiring users to manually search for and select the desired asset each time. With this change, the asset selector now automatically displays the selected element when opened. This improvement streamlines the user workflow by eliminating the need to repeatedly locate and select the same asset, saving time and effort.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55762
+version: 1020.0.0
+---
+Previously, the asset selector did not show the currently selected element by default, requiring users to manually search for and select the desired asset each time. With this change, the asset selector now automatically displays the selected element when opened. This improvement streamlines the user workflow by eliminating the need to repeatedly locate and select the same asset, saving time and effort.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-asset-selector-showing-by-default-the-selected-element-4776.md:17]:
```suggestion
Previously, the asset selector did not show the currently selected element by default, requiring users to always manually search for and select the desired asset. With this change, the asset selector now automatically displays the selected element when opened. This improvement streamlines the user workflow by eliminating the need to repeatedly locate and select the same asset, saving time and effort.
```

**Comment by BeateRixen** [comment]
@rahultsag As this is a new feature, please review and approve the PR.

**Review by rahultsag — APPROVED** [review]

**Pull request merged**